pkg/metadata/host/host_windows.go

```diff
@@ -172,3 +175,46 @@ func Pids() ([]int32, error) {
 	}
 
 }
+
+// GetNetworkInfo returns host specific network configuration.
+// At this time, only information queried is the ephemeral port range
+func GetNetworkInfo() (*NetworkInfo, error) {
+	//
+	// note that this will return the ephemeral port range for ipv4/tcp
+	// Windows allows individual configuration for ipv4/6 and udp/tcp.
+	cmd := exec.Command("netsh", "int", "ipv4", "show", "dynamicport", "tcp")
+	cmdOutput := &bytes.Buffer{}
+
+	// output should be of the format
+	/*
+		Protocol tcp Dynamic Port Range
+		---------------------------------
+		Start Port      : 49000
+		Number of Ports : 16000
+	*/
+	cmd.Stdout = cmdOutput
+	err := cmd.Run()
+	if err != nil {
+		return nil, err
+	}
+	output := cmdOutput.Bytes()
+	var startportline = regexp.MustCompile(`Start.*: (\d+)`)
+	var numberline = regexp.MustCompile(`Number.*: (\d+)`)
```

Comment:
If the codepage is not english, this may panic and/or not find the values I think